crates/binjs_generate_library/src/lib.rs

```diff
@@ -921,8 +921,11 @@ impl<R> Deserializer<R> where R: TokenReader {{
 
 impl<R> InnerDeserialization<R, {name}> for Deserializer<R> where R: TokenReader {{
     fn deserialize_inner(&mut self) -> Result<{name}, R::Error> where R: TokenReader {{
+        print_file_structure!(self.reader, \"{name} {{{{\");
```

Comment:
Yep, that's nicer.

Reply:
The macro is used only in runtime.
I'll keep it inside binjs_io.